source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -500,6 +500,71 @@ network provider on the host (unless implemented by the user).
 * get_caps(caps, properties)
 * get_stats(stats, properties)
 
+## Using OSP8 as an external provider
+This feature aims to integrate with any external network provider. We have tested it with OSP8's Neutron using packstack installation, with the following steps and workarounds.
+
+#### Install OSP8 with packstack on Controller, Compute and Networker(the host that will run the network services)
+Please NOTE: This is how we tested, installed and integrated with oVirt(3.6 and 4.0) step by step.
+Some of the configurations, answers in the answer file and workarounds, may be relevant only to our specific or to some other specific use cases.
+
+### Controller/Neutron VM
+* Create and run rhel 7.2 VM with 2Gb RAM and 40Gb hardisk
+* rpm -i http://rhos-release.virt.bos.redhat.com/repos/rhos-release/rhos-release-latest.noarch.rpm
+* rhos-release 8
+* yum install openstack-packstack
+* packstack --gen-answer-file=$PACKSTACK_ANSWER_FILE
+* Example to our answer file can be found --> [answer_file](http://pastebin.test.redhat.com/360085)
+* In the answer file you can choose and enter the IPs for: Controller, Compute and Networker(the Network host, on which to install the network service such as Compute networking (nova network) or Open Stack Networking)
```

Comment:
=>
Modify the following entries in the answer file:
- controller IP
- compute IP
- networker IP (the Network host, on which to install the network service such as Compute networking (nova network) or Open Stack Networking)